roles/kubernetes-apps/metallb/README.md

```diff
@@ -3,13 +3,15 @@
 MetalLB hooks into your Kubernetes cluster, and provides a network load-balancer implementation.
 In short, it allows you to create Kubernetes services of type "LoadBalancer" in clusters that
 don't run on a cloud provider, and thus cannot simply hook into paid products to provide load-balancers.
-This addon aims to automate [this](https://metallb.universe.tf/concepts/layer2/).
-It deploys MetalLB into Kubernetes and sets up a layer 2 load-balancer.
+This addon aims to automate [MetalLB in layer 2 mode](https://metallb.universe.tf/concepts/layer2/)
+and/or [MetalLB in BGP mode][https://metallb.universe.tf/concepts/bgp/].
```

Comment:
@Miouge1 has a good point.
Here says we can enable both modes with `and/or` word but actually `metallb_protocol` can accept a single protocol only.
It is nice to have more discussion.

/lgtm cancel

Reply:
metallb_additional_address_pools expect a protocol for each additional address pool, so "and/or" is correct as far as kubespray goes, but I've removed both instances of "and/".
Like @EppO said having both at once doesn't really make any practical sense.